tests/run/utf16encoded.flags

```diff
@@ -0,0 +1 @@
+-encoding UTF16
```

Comment:
Yes, you should move it to pos-special and add a line next to https://github.com/lampepfl/dotty/blob/d84805299a42cb8d2c756aff5a117af24dbeaaf4/test/dotc/tests.scala#L113 like:
```scala
@Test def pos_utf16encoded = compileFile(posSpecialDir, "utf16encoded", List("-encoding", "UTF16"))
```